genai-perf/README.md

```diff
@@ -407,13 +407,21 @@ URL of the endpoint to target for benchmarking. (default: `None`)
 
 ##### `-b <int>`
 ##### `--batch-size <int>`
+##### `--batch-size-text <int>`
 
-The batch size of the requests GenAI-Perf should send.
+The text batch size of the requests GenAI-Perf should send.
 This is currently only supported with the
-[embeddings](docs/embeddings.md), image_retrieval, and
+[embeddings](docs/embeddings.md) and
 [rankings](docs/rankings.md) endpoint types.
 (default: `1`)
 
+##### `--batch-size-images <int>`
```

Comment:
nit: do we want to alphabetize these? image then text?

Reply:
I wasn't sure with this. `--batch-size-images` should come before `--batch-size-text`. However, if we look at the doc, the first arg for text is `-b` and the second is `--batch-size`, both of which would come before `--batch-size-images`. Happy to take either approach.

Reply:
Thats a good point that led to a different question.
This is fine, and you clearly considered it already before i asked. 
thanks for being so thorough.